public/app/features/explore/extensions/getExploreExtensionConfigs.tsx

```diff
@@ -38,6 +41,19 @@ export function getExploreExtensionConfigs(): PluginExtensionLinkConfig[] {
           });
         },
       }),
+      createExtensionLinkConfig<PluginExtensionExploreContext>({
+        title: 'Add Correlation',
```

Comment:
we generally use sentence case in the UI
```suggestion
        title: 'Add correlation',
```